pkg/catalog/types.go

```diff
@@ -388,3 +388,56 @@ var (
 	// used by memengine or tae or cn
 	MoTableMetaDefs = []engine.TableDef{}
 )
+
+const QueryResultPath = "s3:/query_result/%s_%s.blk"
```

Comment:
this should be a relative path from/to some file store.   what if we run on localdisk?

Reply:
S3 can be configured to the local disk. 
<img width="1281" alt="截屏2023-01-03 下午5 00 34" src="https://user-images.githubusercontent.com/49111204/210326469-01ab0c4b-af0b-4b16-b259-6c298b5ab528.png">

Reply:
> this should be a relative path from/to some file store. what if we run on localdisk?

S3 file service means the global shared file service. It may be S3 or local disk.

I think it should have been named Shared.